src/routes/webhook/webhook.service.ts

```diff
@@ -110,29 +110,35 @@ export class WebhookService {
               },
             });
           } else if (error.request !== undefined) {
+            const errorMessage = `Response not received. Error: ${error.message}`;
             // Request was made but response was not received
             this.logger.error({
-              message: `Error sending event: Response not received. Error: ${error.message}`,
+              message: 'Error sending event',
               messageContext: {
                 event: parsedMessage,
                 httpRequest: {
                   url: url,
                   startTime: startTime,
                 },
-                httpResponse: null,
+                httpResponse: {
+                  data: errorMessage,
+                },
               },
             });
           } else {
             // Cannot make request
+            const errorMessage = error.message;
             this.logger.error({
-              message: `Error sending event: ${error.message}`,
+              message: 'Error sending event',
               messageContext: {
                 event: parsedMessage,
                 httpRequest: {
                   url: url,
                   startTime: startTime,
                 },
-                httpResponse: null,
+                httpResponse: {
```

Comment:
But this is not accurate. There's no `httpResponse` as `request` couldn't be made. I would propose:

```js 
httpRequest: {
  url: url,
  startTime: startTime,
  error: errorMessage,
}
```

Reply:
I proposed another object to store inside the errors when a response cannot send or is not received. The new object proposed is httpRequestError.